app/experimenter/experiments/models.py

```diff
@@ -452,13 +478,18 @@ def completed_required_reviews(self):
 
     @property
     def completed_all_sections(self):
-        return (
+        completed = (
             self.completed_population
             and self.completed_variants
             and self.completed_objectives
             and self.completed_risks
```

Comment:
Perhaps simpler, instead of requiring a mutating variable, would be:
```python
return (
    self.completed_population
    and self.completed_variants
    and self.completed_objectives
    and self.completed_risks
    and (not self.is_addon_study or self.completed_addon)
)
```

Reply:
Yes that's definitely valid but it seems more a matter of preference.